.github/workflows/update-dependencies.yml

```diff
@@ -29,8 +29,8 @@ jobs:
   cocoa:
     uses: getsentry/github-workflows/.github/workflows/updater.yml@v2
     with:
-      path: modules/sentry-cocoa
-      name: Cocoa SDK (iOS)
+      path: modules/sentry-cocoa.properties
+      name: Cocoa SDK (iOS and Mac)
```

Comment:
Don't we use sentry-native on Mac?

Reply:
No, we switched to using sentry-cocoa on Mac quite a while ago (#190).